interface/views.py

```diff
@@ -76,7 +76,10 @@ def upload(request, course_code, assignment_code):
     else:
         form = UploadFileForm()
 
-    return render(request, 'interface/upload.html', {'form': form})
+    return render(request, 'interface/upload.html', {
+        'form': form,
+        'current_user': request.user,
```

Comment:
This should already be available to all templates as `user`: https://docs.djangoproject.com/en/3.0/ref/templates/api/#django-contrib-auth-context-processors-auth